packages/reference/src/resources/ExternalResourceCard/ExternalResourceCard.tsx

```diff
@@ -27,10 +38,55 @@ const defaultProps = {
   hasCardRemoveActions: true,
 };
 
+const styles = {
+  subtitle: css({
+    color: tokens.gray600,
+    marginBottom: 'none',
+  }),
+  description: css({
+    color: tokens.gray900,
+    marginBottom: 'none',
+    maxWidth: '642px',
+  }),
+};
+
 function ExternalEntityBadge(badge: ExternalResource['fields']['badge']) {
   return badge ? <Badge variant={badge.variant}>{badge.label}</Badge> : null;
 }
 
+function ExternalResourceCardDescription({
+  subtitle,
+  description,
+}: {
+  subtitle?: string;
+  description?: string;
+}) {
+  if (!subtitle) {
+    return null;
+  }
+
+  if (!description) {
+    return (
+      <Paragraph className={styles.description} isWordBreak>
+        {subtitle}
+      </Paragraph>
+    );
+  }
+
+  const truncatedDescription = truncate(description, 500, {});
```

Comment:
can we use <Paragraph isTruncated /> instead?

Reply:
When I talked about it with Hande she wanted a max of 2 ligns on the description and a max width of `642px` that's why I set it like that. using that option will make it way shorter, max of one lign I guess.

Reply:
it still feels weird not using existing apis (`isTruncated`). but not blocking